internal/pkg/metadb/record/record_test.go

```diff
@@ -265,8 +280,10 @@ func TestRecord_NewRecordFromProto(t *testing.T) {
 	createdAt := time.Date(1992, 1, 15, 3, 15, 55, 0, time.UTC)
 	updatedAt := time.Date(1992, 11, 27, 1, 3, 11, 0, time.UTC)
 	proto := &pb.Record{
-		Key:      "key",
-		BlobSize: int64(len(testBlob)),
+		Key:            "key",
+		BlobSize:       int64(len(testBlob)),
+		Chunked:        false,
+		NumberOfChunks: 1,
```

Comment:
I would not populate NumberOfChunks if it's not chunked (the object does not have a chunk). How about using 0 as a placeholder? I think that's the behavior of the server code as well.

Reply:
Good observation.